centers/models/center.py

```diff
@@ -14,8 +14,7 @@ class Center(models.Model):
     address = models.CharField(max_length=255, blank=True, null=True)
     phone = models.CharField(max_length=45, blank=True, null=True)
     url = models.URLField(blank=True, null=True)
-    hash_id = models.CharField(
-        max_length=12, unique=True)
+    hash_id = models.CharField(max_length=12, unique=True)
```

Comment:
I think hash_id is used in url.
so in that case you need to make it using index